.github/workflows/cloud-admin.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and Deploy Cloud Admin to Netlify
+on:
+  push:
+    tags:
+      - "*"
+    branches: ["main"]
+    paths:
+      - ".github/workflows/cloud-admin.yml"
+      - "web-admin/**"
+      - "web-local/**"
```

Comment:
Any reason for `web-local` to be included here?

Reply:
oh it should have been `common`